### PR TITLE
feat: Add knitting option to English

### PR DESCRIPTION
**Pull request by thetriggestcove**

### CODE REVIEW

**Comment by thetriggestcove** [comment]
Please review this, @casey-collab

File: english/marshmallow.md [hunk 1]
```diff
@@ -239,6 +239,8 @@ Do you:
 
 [Blackout and wake up in a strange place without knowing how](33hours/wakingup.md)
 
+Take out your [knitting project](knit/knit.md) and finish your sock heel turn.
```
Comment (casey-collab) [inline english/marshmallow.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by thetriggestcove** [comment]
Please review this, @casey-collab. I fixed the broken link.

**Comment by casey-collab** [comment]
Looks great! :penguin:

**Pull request merged**